apps/admin-gui/src/app/shared/components/dialogs/edit-attribute-definition-dialog/edit-attribute-definition-dialog.component.ts

```diff
@@ -69,17 +69,20 @@ export class EditAttributeDefinitionDialogComponent implements OnInit {
   writeFacility = false;
   pageSize: number;
   tableId = TABLE_ENTITYLESS_ATTRIBUTE_KEYS;
+  loading = false;
 
   ngOnInit() {
+    this.loading = true;
     this.pageSize = this.tableConfigService.getTablePageSize(this.tableId);
     this.dialogRef.addPanelClass('mat-dialog-height-transition');
     this.attDef = this.data.attDef;
     this.serviceService.getServicesByAttributeDefinition(this.attDef.id).subscribe(response => {
       this.services = response;
-    });
-    this.attributesManager.getAttributeRights(this.attDef.id).subscribe(response => {
-      this.fromRightsToCheckboxes(response);
-    });
+      this.attributesManager.getAttributeRights(this.attDef.id).subscribe(attributeRightsResponse => {
+        this.fromRightsToCheckboxes(attributeRightsResponse);
+      });
+      this.loading = false;
```

Comment:
this loading needs to be inside of subscribe of getAttributeRights, because thats also asynchronous operation